StandaloneMmPkg/Core/Dispatcher.c

```diff
@@ -837,7 +837,12 @@ FvIsBeingProcessed (
   DEBUG ((DEBUG_INFO, "FvIsBeingProcessed - 0x%08x\n", FwVolHeader));
 
   KnownFwVol = AllocatePool (sizeof (KNOWN_FWVOL));
-  ASSERT (KnownFwVol != NULL);
+  // MU_CHANGE [BEGIN] - CodeQL change
+  if (KnownFwVol == NULL) {
+    return;

```

Comment:
The assertion removal is intentional?

Reply:
The assert removal was unintentional. I'll add it.

Reply:
Updated in PR commit https://github.com/microsoft/mu_basecore/pull/262/commits/e6aeaab787bff72ff473f9b505d7a94f887bddbe